### ISSUE TITLE
T3 Mage Set Bonus not working

### ISSUE THREAD

**Issue by Zimwik** [dalaranwow/dalaran-wow#5895]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

The T3 Frostfire Regalia set for mages 6-piece set bonus says "Your damage spells have a chance to cause your target to take up to 200 increased damage from subsequent spells". The debuff is called Elemental Vulnerability, but it is not proc'ing at the rate that is Blizzlike. I have tested out spells on test dummies and a variety of mobs for hours and have seen it proc about 3 times (which would make the proc rate significantly below <1%. I have donated in order to buy this set for the purpose that the set bonus would be working.

**Expected behaviour**: Tell us what should happen instead

According to Wowhead and other sources the Blizzlike proc rate should be significantly higher. One source on Wowhead says "The elemental vulnerability has a 20% chance to proc which basically equates to a +40dmg increase. In raids, others will quickly use up this proc but benefitting someone else is, for the most part, the same as benefitting you for overall raid dps." This is definitely not the case currently.

**Steps to reproduce the problem**:

1. Equip at least 6 pieces of the T3 Frostfire Regalia set bonus on a mage.
2. Find any mobs or test dummy to experiment with spells.
3. Test out casting any spells and notice that the Elemental Vulnerability proc almost never occurs, making it an essentially negligible set bonus. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

T3 Frostfire Regalia set: https://classic.wowhead.com/item-set=526/frostfire-regalia
Proof of tested proc rate: https://classic.wowhead.com/spell=28771/elemental-vulnerability

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

Item IDs for T3 Frostfire Regalia Set https://classic.wowhead.com/item-set=526/frostfire-regalia:
Waist ID = 22502
Wrist ID = 22503
Head ID = 22498
Hands ID = 22501
Legs ID = 22497
Chest ID = 22496
Feet ID = 22500
Shoulders ID = 22499
Finger ID = 23062

Elemental Vulnerability Spell Proc ID https://classic.wowhead.com/spell=28771/elemental-vulnerability:
Spell ID = 28771

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

Final note: Feedback and a fix on this issue would be greatly appreciated. Thank you.
